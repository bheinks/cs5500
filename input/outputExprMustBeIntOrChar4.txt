(* 
   Output expression must be of type integer or char
*)

program p;

var A : array [1..100] of boolean;

begin
  write(A[50]);
  write('!')
end.

