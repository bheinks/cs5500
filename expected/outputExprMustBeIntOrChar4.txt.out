Line 10: Output expression must be of type integer or char
